data/human/intro missions.txt

```diff
@@ -413,24 +413,112 @@ mission "Intro [2 Interceptor]"
 			`	"Better start saving up then," he says. "I know a number of old Sparrow pilots who made their first million credits mining asteroids before they could afford a more combat-worthy ship. They had to look around for a while before finding a system with any asteroids worth mining, but it was worth it. Anyway, let's go get your payment." You walk out of the ship and sign the papers to hand off the <commodity> to the dock workers on <planet>. They hand you a credit chip worth <payment>.`
-			`	"If you're willing to take me on one more journey," says James, "meet me in the spaceport bar in a little while."`
+			`	"I've got an idea for something you might want to try," says James. "Meet me in the spaceport bar in a little while."`
 
 
 
 mission "Intro [3 Interceptor]"
+	priority
+	name "Mine at <planet>"
+	description "Mine asteroids at <destination>."
+	passengers 1
+	source "Ingot"
+	destination "New Sahara"
+	to offer
+		has "Intro [2 Interceptor]: done"
+	
+	on offer
+		conversation
+			`James waves you over to his table when you enter the bar. He's sitting with another man with the local star map spread out between them. The steady patter of micrometeorites against the roof reminds you of rain on the factory roof in New Boston.`
+			`	"This is Captain Marshall," James says as you pull up a chair. "He mines in this part of space and says there's a good system nearby if you're willing to give it a try."`
+			choice
+				`	"I'm up for a challenge."`
+					goto mining
+				`	"I'm ready to head out on my own."`
+					decline
+	
+			label "mining"
+			`	"That's good to hear," Captain Marshall says with a smile. "Mining without an asteroid scanner is a test of your piloting skills, but one that pays well."`
+			`	"The first challenge," he continues, "will be finding which asteroids are valuable. The worthless asteroids are either brown and smooth or gray and spiky with no apparent orbit. It doesn't matter how long you shoot them, they won't be destroyed. Valuable asteroids will always orbit the star and have a range of colors and surface textures. Titanium tends to be a bit larger, with a pitted surface. Lead is light gray and has a boxy shape. Silicon has a bit of a greenish cast and looks like it's been melted in places. I increase the resolution on my control displays to make identification and targeting easier.`
+			`	"Also remember that the more valuable the mineral the longer it takes to mine. Silicon breaks up quickly and you'll fill your hold in a hurry. Titanium is more work, but you make several times the number of credits for the effort."`
+			`	Captain Marshall wishes you luck and heads out into the spaceport. You and James spend a few minutes going over the plan as you finish your drinks. "Leave your hold empty and fill it up before you land at <destination>. You can sell what you mine when we land. Remember there's no guaranteed payment in this kind of work. How much you make is based on the value of the minerals in your hold."`
```

Comment:
We'd want to somehow add that you need to mine silicon to continue here. Maybe Marshall says something like "You should probably start easy and bag yourself five tons of silicon."

Reply:
Done: "You should start off easy and mine five tons of silicon. If you've got the hang of it, you can fill your hold with more valuable minerals."